scripts/calculateRevenue.ts

```diff
@@ -6,55 +6,60 @@ import yargs from 'yargs'
 import { protocols, Protocol } from './types'
 
 async function main(args: ReturnType<typeof parseArgs>) {
-  const eligibleAddresses = parse(
-    readFileSync(args['input-addresses'], 'utf-8').toString(),
-    { skip_empty_lines: true },
-  ).flat()
-  const handler = calculateRevenueHandlers[args['protocol-id'] as Protocol]
+  const inputFile = args['input-file'] ?? `${args['protocol']}-referrals.csv`
+  const outputFile = args['output-file'] ?? `${args['protocol']}-revenue.csv`
```

Comment:
nit: yargs should give you these in camel case, so you can do `args.inputFile`, `args.outputFile`, etc which I find a little easier to read.